Makefile

```diff
@@ -33,7 +33,7 @@ deps:
 	docker-compose run --rm cli composer install --no-scripts
 
 composershow:
-	docker-compose run --rm cli bash -c 'composer show --format=json --no-dev --no-ansi --locked | jq ".locked[] | { \"name\": .name, \"version\": .version }" > dependencies.json'
+	docker-compose run --rm cli bash -c 'composer show --format=json --no-dev --no-ansi --locked | jq "[.locked[] | { \"name\": .name, \"version\": .version }]" > dependencies.json'
```

Comment:
Is it square brackets or curly braces?

Reply:
It's correct now. It needed square brackets around the whole thing to tell jq to format the result as an array. After fixing this make command, I used it to regenerate the dependencies.json file, so you can see there the effects of this change.